com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -2997,7 +3000,7 @@ void GetContactShadowMask(HDAdditionalLightData hdAdditionalLightData, BoolScala
             // If contact shadows are not enabled or we already reached the manimal number of contact shadows
             // or this is not rasterization
             if ((!hdAdditionalLightData.useContactShadow.Value(contactShadowEnabled))
-                || m_ContactShadowIndex >= LightDefinitions.s_LightListMaxPrunedEntries
+                || m_ContactShadowIndex >= ShaderConfig.FPTLMaxLightCount
```

Comment:
hmm... I haven't think about this one. Guess contact shadow must stay at 32 max. So we need a separate definition. The contact shadow code write in an RGBA8 render target.

Reply:
Done, made this a nice enum. We keeping it to 24 because we have the fade and bits slots set to 24.
This means that the maximum number of lights that can have contact shadows are 24 per pixel.